kf_lib_data_ingest/app/cli.py

```diff
@@ -267,20 +299,43 @@ def create_new_ingest(dest_dir=None):
     "file_or_dir",
     type=click.Path(exists=True, file_okay=True, dir_okay=True),
 )
-@click.option(
-    "--exclude_implicit",
-    default=False,
-    is_flag=True,
-)
-def validate(file_or_dir, exclude_implicit=False):
-    # TODO
+@click.option(*VALIDATION_MODE_OPT["args"], **VALIDATION_MODE_OPT["kwargs"])
+def validate(file_or_dir, validation_mode=DEFAULT_VALIDATION_MODE):
+    """
+    Validate files and write validation reports to
+    a subdirectory, `validation_results`, in the current working directory
+
+    \b
+    Arguments:
+        \b
+        file_or_dir - the path to the file or directory of files to validate
+    """
     from kf_lib_data_ingest.common.io import path_to_file_list
     from kf_lib_data_ingest.validation.validation import Validator
```

Comment:
Doesn't importing in the middle of a file violate pep8?

Reply:
The clean script didn't fix it..so no? We do this in other places in cli.py so if we change this one, then we should change all of them.